res/shaders/halo.hlsl

```diff
@@ -0,0 +1,56 @@
+cbuffer object_buffer : register(b0)
+{
+    float4x4 projection_view_model;
+    float4x4 world;
+};
+
+cbuffer camera_buffer : register(b2)
+{
+    float3 camera_pos;
+    float padding1;
+    float3 obj_origin;
+    float padding2;
+}
```

Comment:
Putting padding into HLSL cbuffer's is not necessary, but we can keep it so it's simpler to read.

Reply:
Hello, Hubert! Thank you for the insight and a valuable piece of advice. 😍😍😍 I don't mind, we can keep it.

Reply:
Hi Michal! I hope this message finds you well, best regards <3